test/e2e-ocl/main_test.go

```diff
@@ -0,0 +1,17 @@
+package e2e_ocl_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/openshift/machine-config-operator/test/helpers"
+)
+
+func TestMain(m *testing.M) {
+
+	// Ensure required feature gates are set.
+	// Add any new feature gates to the test here, and remove them as features are GAed.
+	helpers.MustHaveFeatureGatesEnabled("OnClusterBuild", "MachineConfigNodes")
```

Comment:
I think we can remove this helper function altogether as we have the featureset specified [in your companion PR](https://github.com/openshift/release/pull/58848/files#diff-f79ad35584926f5ddf68b3b78fe16e1adea24224a90ab826ad448d118854f6f5R224) in openshift/release. I believe we originally added this check in the tech preview suite when feature gate were relatively new and we weren't sure how they should be configured in CI. 

After both PRs merge, and sometime in the not too _distant_ future when the OCL feature gate moves to default, we can finally remove the featureset config for `e2e-gcp-op-ocl` in `openshift/release` as a follow-up PR :smile: